source/shared/cpp/ObjectModel/MarkDownParsedResult.cpp

```diff
@@ -228,7 +229,7 @@ void MarkDownParsedResult::MatchLeftAndRightEmphasises()
                 }
             }
             // check which one has leftover delims
-            (*currentLeftEmphasis)->GenerateTags(*currentEmphasis);
+            m_isHTMLTagsAdded = m_isHTMLTagsAdded | (*currentLeftEmphasis)->GenerateTags(*currentEmphasis);
```

Comment:
>ed | (*cu [](start = 47, length = 9)

Should be logical or not bitwise

Reply:
This code never calls GenerateTags when m_isHTMLTagsAdded is true. The condition should either be flipped or separated into two statements.

---
In reply to: [183227326](https://github.com/Microsoft/AdaptiveCards/pull/1399#discussion_r183227326) [](ancestors = 183227326)

Reply:
thanks for catching this.

---
In reply to: [183489214](https://github.com/Microsoft/AdaptiveCards/pull/1399#discussion_r183489214) [](ancestors = 183489214,183227326)